test_tinypages.py::test_parallel flaky failure on mac

```
FAILED tests/plotting/test_tinypages.py::test_parallel - AssertionError: sphinx build failed with stdout:
  Running Sphinx v8.1.3
  loading translations [en]... done
  Converting `source_suffix = '.rst'` to `source_suffix = {'.rst': 'restructuredtext'}`.
  [autosummary] generating autosummary for: index.rst, some_autodocs.rst, some_plots.rst
  building [mo]: targets for 0 po files that are out of date
  writing output... 
  building [html]: targets for 3 source files that are out of date
  updating environment: [new config] 3 added, 0 changed, 0 removed
  reading sources... [ 33%] index
  
  stderr:
  
  Exception occurred:
    File "/opt/homebrew/Cellar/python@3.10/3.10.20/Frameworks/Python.framework/Versions/3.10/lib/python3.10/multiprocessing/connection.py", line 383, in _recv
      raise EOFError
  EOFError
  The full traceback has been saved in /var/folders/9r/5b5n96017v7bcd_x846p4qwc0000gx/T/sphinx-err-d063lwqr.log, if you want to report the issue to the developers.
  Please also report this if it was a user error, so that a better error message can be provided next time.
  A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
  
  
assert 2 == 0
 +  where 2 = <Popen: returncode: 2 args: ['/Users/runner/gh/runner-5/_work/pyvista/pyvist...>.returncode
```

--- a/tests/plotting/test_tinypages.py
+++ b/tests/plotting/test_tinypages.py
@@ -133,6 +133,7 @@ def test_tinypages(tmp_path: Path, ename: str, evalue: str):
     assert b'This is a matplotlib plot.' in html_contents
 
 
+@flaky_test(exceptions=(AssertionError,))
 @pytest.mark.skip_windows('path issues, e.g. image file not readable')
 @pytest.mark.skip_check_gc
 def test_parallel(tmp_path: Path) -> None:

